diff --git a/lib/cmi.ml b/lib/cmi.ml
index 7334354..a44bcd7 100644
--- a/lib/cmi.ml
+++ b/lib/cmi.ml
@@ -53,7 +53,7 @@ and module_type  = function
     let arg = Option.( mto >>|module_type >>| fun s ->
                        {Arg.name; signature = s} ) in
     Fun { arg ; body = module_type mt}
-  | Mty_alias (_,p) -> M2l.Alias (Paths.Expr.concrete @@ from_path p)
+  | Mty_alias p -> M2l.Alias (Paths.Expr.concrete @@ from_path p)
 
 let m2l path =
   let cm2i = cmi_infos path in
